src/mqttcm_connect.c

```diff
@@ -180,8 +182,6 @@ bool mqttCMConnectBroker()
 			username = clientId;
 			MqttCMInfo("clientId is %s username is %s\n", clientId, username);
 
-			execute_mqtt_script(OPENSYNC_CERT);
```

Comment:
Do we need this macro anymore OPENSYNC_CERT? If not remove it from .h

Reply:
not required, removed